src/Events/Blocks/Controller.php

```diff
@@ -50,25 +50,25 @@ protected function add_actions() {
 	 * @since 6.2.7
 	 */
 	public function remove_actions() {
-		remove_action( 'tribe_editor_register_blocks', [ $this, 'action_register_archive_template' ] );
-		remove_action( 'tribe_editor_register_blocks', [ $this, 'action_register_single_event_template' ] );
+		remove_action( 'tribe_editor_register_blocks', [ $this, 'register_archive_events_block' ] );
+		remove_action( 'tribe_editor_register_blocks', [ $this, 'register_single_event_block' ] );
 	}
 
 	/**
-	 * Registers the Events Archive template.
+	 * Registers the Events Archive block.
 	 *
 	 * @since 6.2.7
 	 */
-	public function action_register_archive_template() {
-		return $this->container->make( Archive_Block_Template::class )->register();
+	public function register_archive_events_block() {
```

Comment:
I know these are "new", but best practice would be to deprecate and return the new function.